addons/website_slides/views/website_slides_templates_lesson_fullscreen.xml

```diff
@@ -4,115 +4,121 @@
-                <div>
-                    <a id="fullscreen_sidebar_button" href="#" class="active o_wslides_fs_toggle_sidebar"><i class="fa fa-bars"></i>Lessons</a>
-                    <a class="o_wslides_small_screen" t-attf-href="/slides/slide/#{slug(slide)}" target="new"><i class="fa fa-pencil"></i>write a review</a>
-                    <div class="o_wslides_fs_slide_title"/>
+            <div class="o_wslides_slide_fs_header d-flex w-100">
+                <a class="p-3 h-100 d-inline-block o_wslides_fs_toggle_sidebar active text-uppercase" href="#">
+                    <i class="fa fa-bars"/> Lessons
+                </a>
+                <a class="p-3 h-100 d-inline-block text-uppercase" t-attf-href="/slides/slide/#{slug(slide)}" target="new">
+                    <i class="fa fa-pencil"/> Write a review
+                </a>
+                <div class="p-3 h-100 d-inline-block o_wslides_fs_slide_title font-weight-bold">
+                    <t>
+                        <t t-call="website_slides.slide_icon">
+                            <t t-set="icon_class" t-value="'mr-2 text'"/>
+                        </t>
+                        <span class="o_wslides_fs_slide_title_span" t-esc="slide.name"/>
+                    </t>
                 </div>
-                <div>
-                    <a t-attf-href="/slides/">back to courses</a>
+                <div class="ml-auto p-3 text-uppercase">
+                    <a t-attf-href="/slides/">Back to courses</a>
                 </div>
             </div>
-            <div class="o_wslides_fs_container">
-                <div class="o_wslides_fs_sidebar">
-                    <ul>
-                        <li>
-                            <div class="o_wslides_fs_sidebar_header">
-                                <a t-attf-href="/slides/#{slug(slide.channel_id)}">
-                                    <span t-field="slide.channel_id.name"/>
-                                </a>
-                                <div class="o_wslides_slide_fs_progress_box">
-                                    <div class="progress o_wslides_fs_sidebar_progressbar">
-                                        <div class="progress-bar" role="progressbar"
-                                            t-att-aria-valuenow="slide.channel_id.completion" aria-valuemin="0" aria-valuemax="100"
-                                            t-att-style="'width: %s' % (slide.channel_id.completion)"/>
-                                    </div>
-                                    <div><span class="o_wslides_progress_percentage" t-esc="slide.channel_id.completion"/>%</div>
-                                </div>
+            <div class="o_wslides_fs_container d-flex flex-wrap">
+                <div class="col-2 o_wslides_fs_sidebar p-0">
+                    <div class="o_wslides_fs_sidebar_header mt-2 p-2">
+                        <a t-attf-href="/slides/#{slug(slide.channel_id)}">
```

Comment:
t-attf is evil ! :smiling_imp: 
We could 't-att' -ize this  and other t-attf from this diff while we're at it